css/store.css

```diff
@@ -0,0 +1,176 @@
+.main{
+    display: flex;
+    width: 100%;
+    height: 1235px;
+    padding-top: 90px;
+
+}
+
+.form{
+    width: 50%;
+    padding: 40px;
+    color: #666666;
+}
+
+.cover{
+    width: 50%;
+    background-repeat: no-repeat;
+    background-image:url(../img/store-image_by-andrew-neel-unsplash.jpg);
+    background-size:cover;
+}
+
+/* FONTS */
+h1{
+    font-weight:500;
+    font-size: 50px;
+    color: var(--orange);
+    padding: 50px 0 59px 0;
+}
+h5{
+    font-size: 18px;
+    color:var(--grey-light);
+    margin-left: 10px;
+}
+
+
```

Comment:
I can see you tried to style and element on multiple occasions like once the font of that then another part for width or colour, or even flex box, I think the best approach would be style an element at once then adjust few relevant parts for an element if requires like for example names, address, postcode and cities etc.